src/components/menu/Portal.tsx

```diff
@@ -0,0 +1,40 @@
+// (C) 2007-2018 GoodData Corporation
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+export interface IPortalProps {
+    targetElement: Element;
+    children: React.ReactNode;
+}
+
+export default class Portal extends React.Component<IPortalProps> {
+    // Thinking with portals!
```

Comment:
what's that?

Reply:
Tomas Reichmann mentioned that it would be a shame if there was no Portal (the game) easter egg :)